css/home.css

```diff
@@ -1,18 +1,71 @@
 /* Home page specific styles */
+html,
+body {
+  margin: 0;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+.footer {
+  background-color: #003c8f;
+  color: white;
+  width: 100%;
+  padding: 40px;
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+  font-family: Roboto;
+  text-align: center;
+  font-size: 16px;
+  line-height: 21px;
+}
+
+.footer__footnote {
+  color: white;
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+}
+
+.footer__footnote::after {
+  content: "";
+  width: 100%;
+  height: 1px;
+  background-color: white;
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: -5px;
+  right: 0;
+  margin: auto;
+}
+
+.footer__icons-container {
+  margin: 30px 5px;
+}
+.footer__icon {
+  margin: 7px;
+}
 
 .homepage-hero {
+  position: absolute;
```

Comment:
why did you add `absolute` here?

Reply:
I dont write that piece of code.

Reply:
please remove it, as gh thinks it is yours

Reply:
done.